### DISCUSSION TITLE
Calendar - Start with unselected date

### DISCUSSION THREAD

**Discussion by bombillazo** [ant-design/ant-design#43040] [Q&A]
Hello, I am using a Calendar component in non-fullscreen mode and it always starts with the current date selected. Is there any way to have the calendar start rendering with no value selected? I tried passing `undefined` to both value and defaultValue, but it did not work.

**Comment by BoyYangzai** [comment]
@bombillazo 
What scenario would require no default value ?
I tried it but seems just NAN-NAN and normal date

**Reply by bombillazo** [reply]
@BoyYangzai  When I am using the calendar as a filter input, and I want the value to start as undefined.

**Comment by Milkywayrules** [comment]
I want the same behavior too, is this isn't a thing until now? Thank you 🙏
